app/models/provider.rb

```diff
@@ -205,33 +216,21 @@ def external_contact_info
       region_code
       telephone
       email
+      website
     ]
 
-    if enrichments.last
-      enrichments.last.attributes.slice(*(attribute_names + %w[website]))
-    else
-      attributes.slice(*attribute_names).merge("website" => url)
-    end
+    attributes.slice(*attribute_names)
   end
 
+  # NOTE: This can be removed, it should not be in use any more
   def content_status
-    newest_enrichment = enrichments.latest_created_at.first
-
-    if newest_enrichment.nil?
-      :empty
-    elsif newest_enrichment.published?
-      :published
-    elsif newest_enrichment.has_been_published_before?
-      :published_with_unpublished_changes
-    else
-      :draft
-    end
+    ""
```

Comment:
We should probably return `:published`

Reply:
Actually, do we even need this method anymore?

Reply:
carded https://trello.com/c/76g6Jm4S/2386-remove-contentstatus-and-lastpublishedat-from-provider-both-mcbe-and-mcfe